src/styles/variables.css

```diff
@@ -89,13 +89,15 @@
   --button-line-height: 2rem;
 
   /*  input */
-  --input-color: inherit;
-  --input-font-size: initial;
-  --input-font-weight: initial;
-  --input-background: #fff;
-  --input-padding: 8px 10px;
+  --input-color: var(--KInputColor, var(--black-70, rgba(0, 0, 0, 0.7)));
```

Comment:
Please do no use and overwrite kongponents variables

If needed - please create a new variable and set value to corresponding value